components/onboarding/Step1.tsx

```diff
@@ -0,0 +1,270 @@
+              phone: values.phone,
+              avatar_url: avatarUrl,
+              year: values.year,
+              fields_of_study: {
+                majors: values.majors,
+                minors: values.minors,
+              },
+              updated_at: new Date(),
+            }, {
+              returning: "minimal", // Don't return the value after inserting
+            })
+            .eq("id", user.id);
+          if (dbError) {
+            setSubmitError(dbError.message);
+          } else {
+            nextStep();
+          }
+          setSubmitting(false);
+        } }
+     >
+        {({ values, setFieldValue, isSubmitting }) => (
+          <Form className="flex flex-col w-1/2 gap-y-4">
+
+            <div>
+              <Field type="text" name="name" placeholder="Name" />
+              <ErrorMessage name="name" component="p" className="text-red-500" />
+            </div>
+
+            <Field type="email" value={ email } disabled />
+
+            <div>
+              <Field type="text" name="username" placeholder="Username" />
+              <ErrorMessage name="username" component="p" className="text-red-500" />
+            </div>
+
+            <div>
+              <Field type="tel" name="phone" placeholder="Phone number (xxx-xxx-xxxx)" />
+              <ErrorMessage name="phone" component="p" className="text-red-500" />
+            </div>
+
+            <div>
+              {values.avatar && (
+                <img
+                  src={ typeof values.avatar === "string" ? values.avatar : URL.createObjectURL(values.avatar) }
+                  className="w-32 h-32 rounded-full m-2 border-black border-2"
+                  alt="Profile"
+                />
+              )}
+              <Dropzone
+                accept="image/jpeg, image/png, image/gif"
+                maxFiles={ 1 }
+                onDrop={ ([file]) => setFieldValue("avatar", file) }
+              >
+                {({ getRootProps, getInputProps }) => (
+                  /* eslint-disable react/jsx-props-no-spreading */
+                  <div { ...getRootProps() } className="p-4 bg-gray-300 border-black border-2 rounded-lg">
+                    <input { ...getInputProps() } />
+                    <p>
+                      Select a profile picture (*.jpeg, *.png, *.gif)
+                      {values.avatar instanceof File && (
+                        <>
+                          :
+                          <b>
+                            {" "}
+                            {values.avatar.name}
+                          </b>
+                        </>
+                      )}
+                    </p>
+                    <ErrorMessage name="avatar" component="p" className="text-red-500" />
+                  </div>
+                )}
+              </Dropzone>
+            </div>
+
+            <div>
+              <Field as="select" name="year">
+                {["", ...Object.keys(Year)].map((year) => (
```

Comment:
Let's add a placeholder for selecting the year.
<img width="205" alt="image" src="https://user-images.githubusercontent.com/33770878/156380244-b7e4edb7-663a-404d-b4c5-d2b54324878e.png">
to
<img width="205" alt="image" src="https://user-images.githubusercontent.com/33770878/156380327-1e7baed1-aaa0-45e4-82e7-d91d25556b1d.png">

with
```js
<Field as="select" name="year">
  <option value="" disabled selected hidden>
    Select your year
  </option>
  {[...Object.keys(Year)].map((year) => (
    <option key={ year } value={ year }>{year}</option>
  ))}
</Field>
  ```

Reply:
:heavy_check_mark: Thanks for the suggestion, that's a nice clean way to do it